src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -218,10 +225,15 @@ private void revokeAccessToCase(User anonymousCaseWorker, String letterHolderId,
         );
     }
 
-    private void updateDefendantId(User defendantUser, String caseId, String defendantId) {
-        coreCaseDataService.update(
+    private CaseDetails updateDefendantId(User defendantUser, String caseId, String defendantId) {
+        String defendantEmail = defendantUser.getUserDetails().getEmail();
```

Comment:
Please add `String defendantEmail` as a argument.

Then set that argument to `defendant@example.com` in uk.gov.hmcts.cmc.claimstore.repositories.support.CCDTestingSupportRepository#linkDefendantToClaim.

Please also rename `updateDefendantId` to `updateDefendantIdAndEmail` or similar.